test/python/tools/test_check_database.py

```diff
@@ -10,25 +10,36 @@
 import pytest
 
 from nominatim.tools import check_database as chkdb
+import nominatim.version
 
 def test_check_database_unknown_db(def_config, monkeypatch):
     monkeypatch.setenv('NOMINATIM_DATABASE_DSN', 'pgsql:dbname=fjgkhughwgh2423gsags')
     assert chkdb.check_database(def_config) == 1
 
 
-def test_check_database_fatal_test(def_config, temp_db):
```

Comment:
This was intentional. the 'temp_db' creates an empty database.

Reply:
ok, removed again